client/src/components/Form.js

```diff
@@ -0,0 +1,170 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+import { isValidMember } from '../lib/validateFunctions';
+
+export default function Form({ submitFunction }) {
+  //Variables
+  const initialMember = {
+    firstName: '',
+    lastName: '',
+    description: '',
+    group: '',
+  };
+
+  const [member, setMember] = useState(initialMember);
+  console.log(member);
+
+  const [isError, setIsError] = useState(false);
+  const [wasSuccessful, setWasSuccessful] = useState(false);
+
+  //Functions
+  function handleChange(event) {
+    const field = event.target;
+    const value = field.value;
+    setMember({ ...member, [field.name]: value });
+  }
+
+  function clickHandler(event) {
```

Comment:
I think in this case clickHandler is too generic and not really telling what the function does. Probably more a submit handler or something formSubmitHandler, as you are more dealing with what happens once the form is submitted